readthedocs/oauth/migrations/0014_remove_remoterepository_project.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.22 on 2021-06-14 15:42
+
+from django.db import migrations
+
+
+def migrate_data(apps, schema_editor):
+    RemoteRepository = apps.get_model('oauth', 'RemoteRepository')
+    queryset = RemoteRepository.objects.filter(project__isnull=False).select_related('project')
+    for rr in queryset:
+        rr.project.remote_repository = rr
+        rr.save()
```

Comment:
@agjohnson this data migration from the old field to the new field is an important change to take a look at carefully.

Reply:
also, this may be slow in production if we have many `RemoteRepositoy` linked to a `Project`.